cluster/ubuntu/build.sh

```diff
@@ -40,7 +40,7 @@ cp flannel-${FLANNEL_VERSION}/flanneld binaries/minion
 
 # ectd
 echo "Download etcd release ..."
-ETCD_VERSION="v2.0.9"
+ETCD_VERSION="v2.0.12"
```

Comment:
We've been running 2.0.12 for a week now: https://github.com/GoogleCloudPlatform/kubernetes/pull/10140